ceph-pr-commits/config/definitions/ceph-pr-commits.yml

```diff
@@ -73,6 +73,7 @@
       - shell:
           !include-raw-verbatim:
             - ../../../scripts/build_utils.sh
+            - ../../../scripts/pr_checks.sh
```

Comment:
This job needs to set the variable `pytest_mark` based on the outcome of the `docs_pr_only` check.

https://github.com/ceph/ceph-build/blob/main/ceph-pr-commits/build/build#L9-L12